app/code/community/Bolt/Boltpay/etc/system.xml

```diff
@@ -297,8 +297,23 @@ This function is called after the checkout form is presented to the user.
               <show_in_default>1</show_in_default>
               <show_in_website>1</show_in_website>
               <show_in_store>1</show_in_store>
-              <sort_order>101</sort_order>
+              <sort_order>103</sort_order>
```

Comment:
Any reason you're changing these sort order values? Maybe just for more flexibility in the future?